example.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Example</title>
+  </head>
+  <body>
+    <script src="./dist/push-notifications.js"></script>
+    <script>
+      var beamsClient = new PusherPushNotifications.PushNotifications({
```

Comment:
This needs to change to new PusherPushNotifications.init().  And `const` is preferred over `var`